creator-node/test/prometheus.test.js

```diff
@@ -25,10 +26,12 @@ describe('test Prometheus metrics', async function () {
     await server.close()
   })
 
-  it('Checks that GET /prometheus_metrics is healthy and exposes default metrics', async function () {
+  it('Checks that GET /prometheus_metrics_worker is healthy and exposes default metrics', async function () {
```

Comment:
I changed it to test /prometheus_metrics_worker prometheus_metrics was broken since our testing suite initializes only one app. the cluster setup is all in index.ts, which is a layer above initializing the app. I think this way of testing one app instance is the correct way to test anyway

Reply:
kk